lib/Notification/Notifier.php

```diff
@@ -91,10 +97,32 @@ public function prepare(INotification $notification, string $languageCode): INot
 		switch ($notification->getSubject()) {
 			case self::SUBJECT:
 				$parameters = $notification->getSubjectParameters();
+				$message = $parameters[0];
 				$notification->setParsedSubject($l->t('Nextcloud announcement'))
-					->setParsedMessage($parameters[0])
 					->setIcon($this->url->getAbsoluteURL($this->url->imagePath($this->appName, 'app-dark.svg')));
 
+				$action = $notification->createAction();
+				$action->setParsedLabel($l->t('Read more'))
+					->setLink($notification->getLink(), IAction::TYPE_WEB)
```

Comment:
Feature is only 17+ so it needs to be removed below, but not sure if the disable button is too confusing then?